sdk/core/azure-core/azure/core/pipeline/policies/_retry.py

```diff
@@ -73,6 +77,8 @@ class RetryPolicy(HTTPPolicy):
 
     :keyword int retry_backoff_max: The maximum back off time. Default value is 120 seconds (2 minutes).
 
+    :keyword RetryType retry_type: Fixed or exponential delay between retries, default is exponential.
```

Comment:
The docstring for `retry_backoff` must be updated to reflect its behavior for fixed backoff...